src/index.html

```diff
@@ -59,16 +59,43 @@ <h1 class="flex flex-wrap">
             id="alert-facebook"
         ></p>
 
-        <div class="wrapper">
-            <form id="form-profile" accept-charset="UTF-8"></form>
-            <p class="text-center mt-5">
-                <button type="button" id="generate-btn" class="btn btn-primary btn-attestation"><span ><i class="fa fa-file-pdf inline-block mr-1"></i>  Générer mon attestation</span></button>
-            </p>
+    <div class="wrapper">
+        <form id="form-profile" accept-charset="UTF-8"></form>
+        <p class="text-center mt-5">
+            <button type="button" id="generate-btn" class="btn btn-primary btn-attestation">
+              <span>
+                <i class="fa fa-file-pdf inline-block mr-1"></i>
+                Générer mon attestation
+              </span>
+            </button>
+        </p>
 
             <div class="bg-primary  d-none" id="snackbar">
                 L'attestation est téléchargée sur votre appareil.
             </div>
+
+          <div id="formgroup-storedata"
+               class="form-check text-center">
+            <input class="form-check-input"
+                   type="checkbox"
+                   name="storedata"
+                   id="field-storedata"
+                   value="storedata">
+            <label class="form-check-label"
+                   for="field-storedata">
+              Stocker mes données sur l'appareil afin de remplir automatiquement mes futures attestations
+            </label>
+          </div>
+
+        <a id="cleardata"
+           class="center delete-data-link"
+           href="javascript:void(0)">Effacer les données du formulaire ?</a>
```

Comment:
Vraiment un petit détail, mais je pense que la ponctuation n'est pas correcte.
Le `?` fait penser que c'est une question amenant l'utilisateur à faire un choix, alors qu'en fait c'est plus une action.
```suggestion
           href="javascript:void(0)">Effacer les données du formulaire</a>
```